unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -20,9 +22,24 @@ def test_sct_label_vertebrae_initz_error():
         sct_label_vertebrae.main(command.split())
 
 
-def test_sct_label_vertebrae_high_value_warning(caplog):
-    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19'
+def test_sct_label_vertebrae_high_value_warning(caplog, tmp_path):
+    command = '-i sct_testing_data/t2/t2.nii.gz -s sct_testing_data/t2/t2_seg-manual.nii.gz -c t2 -initz 40,19 -ofolder ' + str(tmp_path)
     sct_label_vertebrae.main(command.split())
     assert 'Disc value not included in template.' in caplog.text
 
 
+def test_sct_label_vertebrae_clean_labels(tmp_path):
```

Comment:
could you also verify that the output disc labels all exist as expected? (you remember the issue with the dilation of labels, resulting in the discs disappearing)

Reply:
I guess so. It is now done!